kwola/datamodels/BugModel.py

```diff
@@ -44,13 +44,18 @@ class BugModel(Document):
 
     reproductionTraces = ListField(StringField())
 
-    def saveToDisk(self, config, overrideSaveFormat=None, overrideCompression=None):
-        saveObjectToDisk(self, "bugs", config, overrideSaveFormat=overrideSaveFormat, overrideCompression=overrideCompression)
+    def saveToDisk(self, config, subFolderStr, overrideSaveFormat=None, overrideCompression=None):
```

Comment:
I would provide subFolderStr with a default value of None. That way your change to this function does not break any of the places we use saveToDisk in the cloud code